### ISSUE TITLE
Doesn't seem to be working under rails3

### ISSUE THREAD

**Issue by dball** [tardate/authlogic_rpx#24]
I'm trying this under rails3 with odorcicd's rails3 branch of authlogic and it fails as follows when I try to mix in authlogic_rpx:

NameError (uninitialized constant AuthlogicRpx::Session::Config::RPXNow):
/Users/dball/.bundle/ruby/1.8/gems/authlogic_rpx-1.1.1/lib/authlogic_rpx/session.rb:38:in `rpx_key_value'
  /Users/dball/.bundle/ruby/1.8/gems/authlogic_rpx-1.1.1/lib/authlogic_rpx/session.rb:34:in`rpx_key'
  /Users/dball/src/eatrealgood/app/models/user_session.rb:2

Has anyone else gotten this working in rails3?

**Comment by tardate** [comment]
thanks for the info @dball. I've had one report of someone finding 'it just works' under  rails 3, but haven't tested yet myself. On the list of things to do...

**Comment by woto** [comment]
I think the problem associated with bundler. I also had this error but in rails 2.3.8
Sorry for bad English

**Comment by woto** [comment]
hmmm, the problem was in rpx_now, when i've created folder rails with init.rb file it's works

**Comment by grosser** [comment]
did you try to require 'rpx_now' ?

**Comment by woto** [comment]
after making this

```
rm -rf ~/.bundle/ ~/.gem/ .bundle/ Gemfile.lock
bundle install
```

erorrs gone

**Comment by tardate** [comment]
rails3 compatibility included in 1.2.0 release
